smithy-aws-protocol-tests/model/restJson1/http-response-code.smithy

```diff
@@ -73,3 +95,29 @@ apply HttpResponseCode @httpResponseTests([
         appliesTo: "client"
     },
 ])
+
+apply ResponseCodeRequired @httpResponseTests([
+    {
+        id: "RestJsonHttpResponseCodeRequired",
+        documentation: """
+                This test ensures that servers handle @httpResponseCode being @required.""",
+        protocol: restJson1,
+        code: 201,
+        body: "",
+        params: {"responseCode": 201}
```

Comment:
Comma missing after the `}` causes CI to fail on the invalid model.

Reply:
Fixed, thanks!